app/src/main/java/org/mozilla/fenix/components/BackgroundServices.kt

```diff
@@ -193,9 +180,49 @@ class BackgroundServices(
     ).also {
         it.registerForDeviceEvents(deviceEventObserver, ProcessLifecycleOwner.get(), false)
```

Comment:
UnreIated comment to what you're changing here:
you currently always register for device events, even if sendTab is disabled. That doesn't seem correct, as it means we'll be able to receive events (if we poll for them) even when sendTab isn't enabled.

Reply:
Easy change to fix!